### ISSUE TITLE
no update on maven

### ISSUE THREAD

**Issue by lcottret** [sbmlteam/jsbml#232]
Dear Jsbml team,

Thanks for updating log4j on the master ! But, the problem is that the package in maven has not changed since 2020. Do you plan to update the maven package so that it includes all the vulnerability fixes ?

Regards,
Ludo Cottret

**Comment by lcottret** [comment]
Another idea, already mentioned in the issues : use another library then log4j that causes a lot of problems when we integrate it in other packages...

**Comment by draeger** [comment]
Yes, that is correct. We should create a new release very soon! Thanks for pointing that out.

**Comment by draeger** [comment]
A new snapshot release is available from [Maven](https://oss.sonatype.org/content/repositories/snapshots/org/sbml/jsbml/) now. A new stable release will follow.

**Comment by draeger** [comment]
To use the current snapshot, please use
```
<dependency>
  <groupId>org.sbml.jsbml</groupId>
  <artifactId>jsbml</artifactId>
  <version>1.6-SNAPSHOT</version>
</dependency>
```

**Comment by lcottret** [comment]
Great ! Thanks.

**Comment by lcottret** [comment]
Note : I had to add this to my pom.xml to find the dependency : 
<repositories>
        <repository>
            <id>dependency-snapshots-repo</id>
            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
            <releases>
                <enabled>false</enabled>
            </releases>
            <snapshots>
                <enabled>true</enabled>
            </snapshots>
        </repository>
    </repositories>

**Comment by lcottret** [comment]
I hope that the stable release will follow soon : we can't publish our package in maven central with non released dependencies. Maybe for Christmas day :-) ?

**Comment by lcottret** [comment]
Hi,
Happy new year !
I repeat my request: could you publish the current release to the maven central repository please ?

**Comment by niko-rodrigue** [comment]
> Hi, Happy new year ! I repeat my request: could you publish the current release to the maven central repository please ?

Happy new year, yes we are doing that at the moment. Trying to setup @draeger so that he can do the full release cycle and knows all the steps.
There is currently an error when uploading to maven central that is not present when doing snapshot, we are looking into it. Hopefully, the release should be done this week. We will update the ticket as soon as it is ready to try.

**Comment by draeger** [comment]
@lcottret I am working on it. There was an error message that we could not solve yet. The problem is in one of the many SBML extension packages that could not successfully be uploaded to Maven (we don't know yet why that is happening) but since the processs is transactional it can only be completed when the entire package is successfully submitted. In other words, one package crashes everything. We are looking for the error as we speak.

**Comment by lcottret** [comment]
Thanks @niko-rodrigue and @draeger . I sympathize with you, publishing in maven central is often a nightmare !

**Comment by draeger** [comment]
This is now solved with [release 1.6.1](https://github.com/sbmlteam/jsbml/releases/tag/jsbml-1.6.1).

**Comment by lcottret** [comment]
Thanks for this new release !
